Microsoft.PowerShell.WhatsNew/relnotes/What-s-New-in-PowerShell-71.md

```diff
@@ -71,30 +71,30 @@ While not officially supported, the community has also provided packages for
 
 ## Experimental Features
 
-For more information about the Experimental Features, see [Using Experimental Features](../learn/experimental-features.md).
+For more information about the Experimental Features, see [Using Experimental Features](/powershell/scripting/learn/experimental-features).
```

Comment:
Since we can update now, should we include content in the module or require that the user run `Update-WhatsNew` after install?

If we remove the content then I need to add some error handling in `Get-WhatsNew`.

Reply:
I think `Get-WhatsNew` should do an update automatically the first time.  I think we can assume this module isn't used in offline scenarios.  However, that can be a separate PR.